Gruntfile.js

```diff
@@ -166,8 +70,11 @@ module.exports = function(grunt) {
             js: {
                 // Files to be uploaded.
                 upload: [{
-                    src: 'dist/smooch.min.js',
+                    src: 'dist/smooch.js',
                     dest: 'smooch.min.js'
+                },{
+                    src: 'dist/smooch.js.map',
```

Comment:
makes it easier to debug for people who integrates it and we can get better error output from people who report errors.